group_vars/all/main.yml

```diff
@@ -163,3 +163,18 @@ network_policy_rule_ingress_controller:
       matchLabels:
         "app.kubernetes.io/component": controller
         "app.kubernetes.io/name": ingress-nginx
+
+
+# UID to use for running containers on k8s.
+#
+# It will be used in k8s templates to specify the SecurityContext.RunAsUser
+# field, to force pods to run with this specific UID.
+#
+# It prevents from running a container as root.
+#
+# We recommend that you specify an unique UID per customer or environment.
```

Comment:
```suggestion
# We recommend that you specify a unique UID per customer or customer/environment.
```

Reply:
Why not emulate openshift by running on random uids by default?

Reply:
In fact, Openshift generates a random UID range per namespace and allows us to use all UIDs of this range.
But according to [their documentation](https://www.openshift.com/blog/a-guide-to-openshift-and-uids) :

> When a Pod is deployed into the namespace, by default, OpenShift will use the first UID and first GID from this range to run the Pod.

And since we don't specify manually UIDs for our pods, in fact, we are always using the same UID per namespace.
And the solution proposed in this PR is the same, IMO.